templates/shared/forms/interactive/storage.html

```diff
@@ -4,263 +4,42 @@
-            <label class="p-radio">
-              <input class="p-radio__input" type="radio" aria-labelledby="storage-Yes" name="redundant-storage" value="Yes">
-              <span class="p-radio__label" id="storage-Yes">Yes</span>
-            </label>
-            <label class="p-radio">
-              <input class="p-radio__input" type="radio" aria-labelledby="storage-No" name="redundant-storage" value="No">
-              <span class="p-radio__label" id="storage-No">No</span>
-            </label>
-            <label class="p-radio">
-              <input class="p-radio__input" type="radio" aria-labelledby="storage-Dont-know" name="redundant-storage" value="I don't know">
-              <span class="p-radio__label" id="storage-Dont-know">I don't know</span>
-            </label>
-          </div>
-        </div>
-      </div>
+    <div class="js-pagination js-pagination--1">
       <div class="row u-no-padding">
         <div class="u-sv3 js-formfield">
           <h3 class="p-heading--5">Tell us about your project and team</h3>
-          <textarea id="about-your-project-or-interest" name="sector-specific-requirements" aria-label="Tell us about your project and team" placeholder="Where are you located? Do you already have OpenStack? Is this initiative for a specific workload or for the business as a whole? Do you have particular regulatory constraints or priorities?" rows="3"></textarea>
+          <p>For example, where are you located? Is this for a specific workload or for the business as a whole? How much data are you planning on storing, and will there be ongoing growth? Do you know what level of performance you need? Do you have regulatory constraints or priorities?</p>
+          <textarea id="about-your-project-or-interest" name="sector-specific-requirements"
+            aria-label="Tell us about your project and team"
+            placeholder=""
+            rows="3"></textarea>
         </div>
       </div>
-      <div class="pagination">
-        <a class="pagination__link--previous p-button" href="">Previous</a>
-        <a class="p-button--positive pagination__link--next" href="">Next</a>
-      </div>
-    </div>
 
-    <div class="js-pagination js-pagination--3 u-hide">
-      <h3 class="p-heading--5">How should we get in touch?</h3>
       <div class="row u-no-padding">
-        <div class="col-6">
-          <form action="/marketo/submit" method="post" id="mktoForm_%% formid %%" class="modal-form">
-            <ul class="p-list">
+
+        <form action="/marketo/submit" method="post" id="mktoForm_%% formid %%" class="modal-form">
+          <div class="row">
+            <ul class="p-list col-6">
               <li class="p-list__item">
                 <label for="firstName">First name:</label>
-                <input required id="firstName" name="firstName" maxlength="255" type="text" required="required" />
+                <input id="firstName" name="firstName" maxlength="255" type="text" required="required" />
```

Comment:
I think the `required="required"` at the end should also be removed:
```suggestion
                <input id="firstName" name="firstName" maxlength="255" type="text" />
```